### DISCUSSION TITLE
Is it thoroughly tested on wide strings?

### DISCUSSION THREAD

**Discussion by martinsolovey** [nlohmann/json#3264] [Q&A] [answered]
On my application I handle very dynamic paths that can contain any kind of weird chars from all kind of languajes, would like to know if I can rest quietly knowing that the lib is gonna have no trouble with that, hope somebody can comment on it ♥

**Comment by gregmarr** [comment, marked as answer]
As long as they're encoded in UTF-8, you should be fine.

**Reply by nlohmann** [reply]
Exactly. The library only supports UTF-8, and is extensively tested with it.
